source/game.production.productionconcept.bmx

```diff
@@ -1212,18 +1213,18 @@ endrem
 
 
 			'POINTS ADD TO TIME !
-			local focusPoints:int = productionFocus.GetFocusPointsSet()
-			'ignore points without penalty
-			focusPoints :- (teamPoints + speedPoints)
-			if focusPoints > 0
-				For local i:int = 0 until focusPoints
-					if script.IsFictional()
-						base :+ (15 + i*7) * TWorldTime.MINUTELENGTH
-					else
-						base :+ (10 + i*5) * TWorldTime.MINUTELENGTH
-					endif
-				Next
-			endif
+			For Local focusIndex:Int = EachIn productionFocus.activeFocusIndices
+				If focusIndex <> TVTProductionFocus.TEAM And focusIndex <> TVTProductionFocus.PRODUCTION_SPEED
+					Local focusPoints:Int = productionFocus.GetFocus(focusIndex)
+					For Local i:Int = 0 until focusPoints
+						if script.IsFictional()
+							base :+ (45 * (i+1) + 3 * 2^i) * TWorldTime.MINUTELENGTH
```

Comment:
Gerne kann hier jetzt der Unterschied zwischen fiktionalem Programm und nichtfiktionalem Programm groesser ausfallen.

"Nichtfiktional" bedeutet ja man kann mit der Kamera mehrere Takes machen, aber nimmt an sich kein "Schauspiel" auf (Ausnahme vlt "Einspieler" die Dinge nachstellen). Beim fiktionalen braucht es evtl mehr Takes, mehr Aufwand um "Technik" zu verstecken (Multikameraaufnahmen usw.).
also 45 vs 25 oder gar 20?

Reply:
Ich würde es tatsächlich erstmal so lassen. Aktuell gibt es in der Datenbank gar keine fiktionalen Drehbücher (die Konfigurationsmöglichkeit war nicht verwendet, Flag wird beim Datenbankeinlesen nicht gesetzt; daher habe ich das nicht dokumentiert und bin erst an diesem Punkt drüber gestoßen).

Bei einer Datenbanküberarbeitung inklusive Auswertung würde ich es nicht "fictional" nennen wollen, da es in allen anderen Fällen die Bedeutung "ausgedacht" vs. "real" hat, auf Drehbücher übertragen also eher "ausgedachtes Drehbuch" vs. "Drehbuch für einen Film, den es wirklich gibt".

Reply:
OK. Ich war neben der Spur und hatte nicht gesehen, dass schon Logik dahinter steht.

Ich schau' mir den Unterschied zwischen den beiden Varianten nochmal an...